packages/components/board/stories/board.stories.js

```diff
@@ -5,14 +5,19 @@ import '@tradeshift/elements.board';
 
 storiesOf('ts-board', module)
 	.addDecorator(withKnobs)
-	.add('default', () => {
-		const dir = text('ltr', 'Dir');
-		const title = text('My Board', 'Title');
-		const content = text('Put any html inside the board', 'Content');
-
-		return html`
-			<ts-board dir="${dir}" data-title="${title}">
-				${content}
-			</ts-board>
-		`;
-	});
+	.add(
+		'default',
+		() => {
+			const dir = text('ltr', 'Dir');
+			const title = text('My Board', 'Title');
```

Comment:
I think the label and value are in wrong order. Another think for sake of consistency and clarity, I think we should use the attribute for the label, so in here it should be `data-title` instead of 'Title'. What do you think?